server/RenderContext.cs

```diff
@@ -1757,5 +1806,111 @@ private void DrawMicroBorders(BorderLayer layer)
                 }
             }
         }
+
+        /// <summary>
+        /// Draw the regions
+        /// </summary>
+        /// <param name="layer">The layer</param>
+        private void DrawRegions(BorderLayer layer)
```

Comment:
How is this different from DrawBorders?

Do we need a distinct type, or could we just have Regions be Borders w/ Style = `None` ?